docs/source/user/parallel.rst

```diff
@@ -84,7 +84,11 @@ function/operator using its previous value in the loop body. The initial value
 of the reduction is inferred automatically for ``+=`` and ``*=`` operators.
 For other functions/operators, the reduction variable should hold the identity
 value right before entering the ``prange`` loop.  Reductions in this manner
-are supported for scalars and for arrays of arbitrary dimensions.
+are supported for scalars and for arrays of arbitrary dimensions. Care should be
```

Comment:
I think it would be better if this section on what not to do was moved to right before the negative example you included below.